src/natcap/invest/utils.py

```diff
@@ -583,8 +583,12 @@ def read_csv_to_dataframe(
     # allow encoding kwarg to override this if it's provided
     if not encoding and has_utf8_bom(path):
         encoding = 'utf-8-sig'
-    dataframe = pandas.read_csv(path, engine=engine, encoding=encoding,
-                                sep=sep, **kwargs)
+    try:
+        dataframe = pandas.read_csv(
+            path, engine=engine, encoding=encoding, sep=sep, **kwargs)
+    except UnicodeDecodeError as error:
+        LOGGER.error(f'{path} must be encoded as utf-8')
```

Comment:
I don't know if it's worth mentioning that other encodings are fine as long as they only use the ASCII-compatible subset of characters? Maybe that wouldn't be any help to most users